index.html

```diff
@@ -141,6 +164,40 @@ <h2 class="head-description1">Profesional Art Printing Data</h2>
                 See Project
             </button>
         </section>
+        <section class="content2">
+            <div>
+                <h2 class="head-description1">Data Dashboard Healthcare</h2>
+                <p class="project-description1">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required. Has 
+                    been the industry's standard.</p>
+                <ul class="programming-language">
+                    <li class="item1">CSS</li>
+                    <li class="item1">HTML</li>
+                    <li class="item1">Ruby</li>
+                    <li class="item1">Bootstrap</li>
+                </ul>
+            </div>
+            <button class="button-project1">
+                See Project
+            </button>
+        </section>
+        <section class="content3">
+            <div>
+                <h2 class="head-description1">Website Portfolio</h2>
+                <p class="project-description1">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required. Has 
+                    been the industry's standard.</p>
+                <ul class="programming-language">
+                    <li class="item1">CSS</li>
+                    <li class="item1">HTML</li>
+                    <li class="item1">Ruby</li>
+                    <li class="item1">Bootstrap</li>
+                </ul>
+            </div>
+            <button class="button-project1">
+                See Project
+            </button>
+        </section>
```

Comment:
kindly add a button for only the first card and also make the card to be center aligned to give the project section a better alignment required and useful 😄 

Figma design : 
![image](https://github.com/AndresEspi/portafolio/assets/20537712/eb456de3-d8e4-44c6-8cd0-942796402564)

Your design :
![image](https://github.com/AndresEspi/portafolio/assets/20537712/eb23b828-c731-4ceb-8739-e48fd34ec1fc)